src/main/java/com/exasol/adapter/installer/VirtualSchemaProfile.java

```diff
@@ -0,0 +1,92 @@
+package com.exasol.adapter.installer;
+
+import java.util.List;
+
+/**
+ * Virtual Schema Profile is dialect-based information need to install a Virtual Schema.
+ */
+public interface VirtualSchemaProfile {
```

Comment:
it seems to me that most of the methods of this interface belong to the `UserInput` class.
I would suggest to move them

Reply:
We don't want to collect information from multiple places, this interface does aggregation